# HG changeset patch
# User Rik <rik@octave.org>
# Date 1450628398 28800
#      Sun Dec 20 08:19:58 2015 -0800
# Node ID 6982def1d416d3a44895fa6d0e1a2cfed764cdd7
# Parent  758b71e964ce2b048dba62e89a08758944812146
maint: Delete unnecessary nargin variable from __ichol0__.

* __ichol__.cc (__ichol0__): Delete unnecessary nargin variable.

diff --git a/libinterp/corefcn/__ichol__.cc b/libinterp/corefcn/__ichol__.cc
--- a/libinterp/corefcn/__ichol__.cc
+++ b/libinterp/corefcn/__ichol__.cc
@@ -180,20 +180,18 @@ void ichol_0 (octave_matrix_t& sm, const
 
 DEFUN (__ichol0__, args, ,
        "-*- texinfo -*-\n\
 @deftypefn  {} {@var{L} =} __ichol0__ (@var{A})\n\
 @deftypefnx {} {@var{L} =} __ichol0__ (@var{A}, @var{michol})\n\
 Undocumented internal function.\n\
 @end deftypefn")
 {
-  int nargin = args.length ();
-
   std::string michol = "off";
-  if (nargin == 2)
+  if (args.length ())
     michol = args(1).string_value ();
 
   // In ICHOL0 algorithm the zero-pattern of the input matrix is preserved
   // so it's structure does not change during the algorithm.  The same input
   // matrix is used to build the output matrix due to that fact.
   octave_value_list param_list;
   if (! args(0).is_complex_type ())
     {
